### PR TITLE
1609218: Add ping reason field in ping_info

### PR DESCRIPTION
**Pull request by mdboom**
Checklist for reviewer:

- [x] Commits should reference a bug or github issue, if relevant
- [x] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [x] Trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional)

For glean changes:
- [x] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

**Comment by jklukas** [comment]
Do we need to update the glean changelog about this addition?

**Comment by mdboom** [comment]
> Do we need to update the glean changelog about this addition?

Indeed.  Thanks for the reminder.

File: templates/include/glean/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Version 1
 
+- An optional `reason` field has been added to `ping_info`.
```
Comment (jklukas) [inline templates/include/glean/CHANGELOG.md]:
```suggestion
- An optional `reason` field has been added to `ping_info`. See [1609218](https://bugzilla.mozilla.org/show_bug.cgi?id=1609218)
```

**Comment by jklukas** [comment]
I'll merge once integration tests finish.

**Pull request merged**